week3/intro.txt

```diff
@@ -1,6 +1,6 @@
 My five favorite things are:
-- 
-- 
-- 
-- 
-- 
+- Music
+- Video Games
```

Comment:
The only video game I play is FIFA. What games are you currently playing and which one is your favorite?  Can you add them to the list?